vpr/src/base/echo_files.h

```diff
@@ -12,6 +12,9 @@ enum e_echo_files {
     E_ECHO_PRE_PACKING_MOLECULES_AND_PATTERNS,
     E_ECHO_VPR_CONSTRAINTS,
 
+    //Packing
```

Comment:
At top, file controls echo files, we have an enum to each one, search for in code to see what each one prints

Reply:
say what echo files are, what they are used for - dumps that intended to be written and read by a developer, typically direct reps of data structures to make debugging easier

Reply:
enum e_output_files - line 66 - see what these are in code and leave a brief comment, or make an issue saying this may just be same as other enum and we should just put them both together

Reply:
set an issue to clean up these two enums (either just need to comment, or merge the two diff enums)
whole thing could be wrapped in a class

Reply:
echo file name can maybe also take a default name (person can choose to pass it in - if not, default set to null), the set_output_filename would work fore everything

investigate - is there some other reason to have the second output enum? - look at the call sites for that

in the cpp file -> instead of the mallocs, recode using c++ strings and wrap it all in a class 

-> all the members of the class could be static members (don't have to turn into class, see if it's worth it) (make a member object of the echo class in something like the vpr_options file)

- might be better to not have class, either way clean up the code by getting rid of mallocs and use c++ strings